README.md

```diff
@@ -83,13 +83,29 @@ It is recommended to end the path component of the URL with `/graphql`, for exam
 
 # Serialization Format
 
-Similar to the GraphQL specification this specification does not require a specific serialization format.
-For consistency and ease of notation, examples of the response are given in JSON throughout the spec.
+The GraphQL specification allows for many [serialization formats to be implemented](https://spec.graphql.org/June2018/#sec-Serialization-Format). In practice JSON is the most common serialization format. In order to be compliant, servers and clients implementing GraphQL over HTTP must support JSON serialization.
```

Comment:
Wholeheartedly agree with that change. It is really important for interoperability to have a common serialization format.

Maybe we can change the text slightly to clarify that supporting JSON is a minimal requirement, but usage of other formats is also fine.

Reply:
```suggestion
The GraphQL specification allows for many [serialization formats to be implemented](https://spec.graphql.org/June2018/#sec-Serialization-Format). Servers and clients over HTTP MUST support JSON and MAY support other, additional serialization formats.
```